starboard/raspi/shared/platform_configuration/configuration.gni

```diff
@@ -12,26 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import("//starboard/build/config/base_configuration.gni")
+if (current_toolchain == default_toolchain &&
+    build_with_separate_cobalt_toolchain) {
+  import(
+      "//starboard/evergreen/arm/softfp/platform_configuration/configuration.gni")
 
-arm_float_abi = "hard"
+  platform_tests_path = "//starboard/raspi/shared:starboard_platform_tests_install($starboard_toolchain)"
+  cobalt_font_package = "standard"
+} else {
+  import("//starboard/build/config/base_configuration.gni")
 
-sb_static_contents_output_data_dir = "$root_out_dir/content"
+  arm_float_abi = "hard"
 
-no_pedantic_warnings_config_path =
-    "//starboard/raspi/shared/platform_configuration:no_pedantic_warnings"
-pedantic_warnings_config_path =
-    "//starboard/raspi/shared/platform_configuration:pedantic_warnings"
-sabi_path = "//starboard/sabi/arm/hardfp/sabi-v$sb_api_version.json"
+  sb_static_contents_output_data_dir = "$root_out_dir/content"
 
-platform_tests_path = "//starboard/raspi/shared:starboard_platform_tests"
+  no_pedantic_warnings_config_path =
+      "//starboard/raspi/shared/platform_configuration:no_pedantic_warnings"
+  pedantic_warnings_config_path =
+      "//starboard/raspi/shared/platform_configuration:pedantic_warnings"
+  sabi_path = "//starboard/sabi/arm/hardfp/sabi-v$sb_api_version.json"
 
-install_target_path = "//starboard/raspi/shared/install_target.gni"
+  platform_tests_path = "//starboard/raspi/shared:starboard_platform_tests_install($starboard_toolchain)"
+
+  speed_config_path = "//starboard/raspi/shared/platform_configuration:speed"
+  size_config_path = "//starboard/raspi/shared/platform_configuration:size"
 
-speed_config_path = "//starboard/raspi/shared/platform_configuration:speed"
-size_config_path = "//starboard/raspi/shared/platform_configuration:size"
+  # TODO(b/219073252): Enable -fno-exceptions and don't mix it with -fexceptions.
```

Comment:
dunno if @y4vor has comments.
Again feel free to resolve and submit

Reply:
This hasn't changed, the diff is just a little off